PA.py

```diff
@@ -1,685 +1,700 @@
-            fun_talk(f'Your upload speed is {round(up_speed, 3)} Mbps')
-
-        elif 'send message on whatsapp' in query:
-            phno_list = {
-                'Umesh': '+911234567890',
-                'Rishabh': '+919876543210',
-                'assitant': '+910000000000'
-            }
-
-
-            def send_whtmsg():
-                fun_talk('To whom you want to send message on WhatsApp')
-                recepient = get_command()
-                check_recep = phno_list[recepient]
-                print(check_recep)
-                fun_talk('Tell me the text in your message')
-                msg = get_command()
-                fun_talk('Do you want to send it immediately?')
-                act_msg = get_command()
-                if 'yes' in act_msg:
-                    hr = datetime.datetime.now().time().hour
-                    min = datetime.datetime.now().time().minute
-                    pywhatkit.sendwhatmsg(check_recep, msg, hr, min + 2)
-                    print('Hey lazy person. Your message is sent successfully.')
-                else:
-                    fun_talk('At what time you want to send this message. For example, 11:21 PM')
-                    msg_time = get_command()
-
-                    hr = 12
-                    min = 52
-                    pywhatkit.sendwhatmsg(check_recep, msg, hr, min)
-                    print('Hey lazy person. Your message is sent successfully.')
-                fun_talk('Do you want to send more WhatsApp messages?')
-                more_msg = get_command()
-                if 'yes' in more_msg:
-                    send_whtmsg()
-
-
-            send_whtmsg()
-
-        elif 'how to' in query:
-            try:
-                # query = query.replace('how to', '')
-                max_results = 1
-                data = search_wikihow(query, max_results)
-                # assert len(data) == 1
-                data[0].print()
-                fun_talk(data[0].summary)
-            except Exception as e:
-                fun_talk('Sorry, I am unable to find the answer for your query.')
-                     
-        elif 'news' in query or 'news headlines' in query:
-            url = "https://news.google.com/news/rss"
-            client = urlopen(url)
-            xml_page = client.read()
-            client.close()
-            page = bs4.BeautifulSoup(xml_page, 'xml')
-            news_list = page.findAll("item")
-            fun_talk("Today's top headlines are--")
-            try:
-                for news in news_list:
-                    print(news.title.text)
-                    # print(news.pubDate.text)
-                    fun_talk(f"{news.title.text}")
-                    # fun_talk(f"{news.pubDate.text}")
-                    print()
-
-            except Exception as e:
-                pass
+#all imports are listed here
+import pyttsx3      
+from PIL import Image                               # pip install pyttsx3

```

Comment:
add your imports at the end of other imports, same for the code of that functionality
```suggestion
from PIL import Image                               
```